uint8 can_id
uint8 script_number
---
string result
